#include <iostream>
using namespace std;
int main(){
    //CPP Handling errors using exceptions
    
    return 0;
}